openapi.yaml

```diff
@@ -4997,7 +4997,8 @@ paths:
 
         **Note:** Next Generation Network (NGN) data centers do not support IPv6 `/116` pools or IP Failover.
         If you have these features enabled on your Linode and attempt to migrate to an NGN data center,
-        the migration will not initiate. NGN data centers include Toronto and Mumbai.
+        the migration will not initiate. NGN data centers include Toronto and Mumbai. If a Linode cannot be
+        migrated because of an incompatibility, the user will be prompted to contact support.
```

Comment:
The error message also suggests choosing a different datacenter, so maybe "the user will be prompted to select a different data center or to contact support"?